### PR TITLE
Add basic UI design

### PR DESCRIPTION
**Pull request by arashnrim**
This PR will add some basic UI on the following pages/places:

- `/cards/[id]`;
- `/cards/`;
- `/log-in/`; and
- everywhere that uses `h1`, `h2`, and `button`.

Please take a brief look at it and let me know if there's anything I can do better! Of course, we can take this design and improve it too as we go on.

Thank you! 🙌

### CODE REVIEW

**Review by exu3 — APPROVED** [review]
I think this looks great!

File: styles/globals.css [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+    h1 {
+        @apply text-3xl font-bold lg:text-5xl;
+    }
+
+    h2 {
+        @apply text-xl font-bold lg:text-3xl;
+    }
+
+    button {
+        @apply px-3 py-1 !bg-red-200 rounded-md shadow-red-200 shadow-sm hover:shadow-red-200 hover:shadow-md transition-shadow;
```
Comment (maggie-j-liu) [inline styles/globals.css:15]: yayy tailwind has colored shadows now

**Review by maggie-j-liu — APPROVED** [review]
this looks awesome!

File: styles/globals.css (hunk 1 above)
Comment (arashnrim) [reply styles/globals.css:15]: Yep! I knew I had to get my hands dirty and use it as soon as I could, it's about time! 😆

**Pull request merged**